Ladybird/AppKit/UI/LadybirdWebView.mm

```diff
@@ -180,6 +180,24 @@ - (void)handleVisibility:(BOOL)is_visible
     m_web_view_bridge->set_system_visibility_state(is_visible);
 }
 
+- (void)findInPage:(NSString*)query
+{
+    auto* paste_board = [NSPasteboard pasteboardWithName:NSPasteboardNameFind];
+    [paste_board setString:query forType:NSPasteboardTypeString];
+
+    m_web_view_bridge->find_in_page(Ladybird::ns_string_to_string(query));
+}
+
+- (void)findInPageNextMatch
+{
```

Comment:
doesn't have to be in this PR, but for other mac apps you can select some text in one app and hit cmd-e to copy it in the find pasteboard (in TextEdit, say), and then switch to another app (e.g. safari) and hit cmd-g to search for what's in the find pasteboard.

while searching in ladybird now lets us cmd-g in a different app and search for the same thing we searched for in ladybird, the other direction doesn't work with the current impl.

Reply:
Ah that's way more featureful than I thought - I think I've gotten it all working like Safari
* cmd+f will open the search panel, and start a search with whatever is in the find pasteboard (if anything)
* typing in the search panel will copy to the find pasteboard
* cmd+g will operate from the pasteboard (starting a new search if our current text differs from the pasteboard)
* cmd+e will copy the selected text to the pasteboard, and if the search panel is open, will start a search with that text